cmd/status/status.go

```diff
@@ -44,34 +42,44 @@ func NewStatusCmd() *cobra.Command {
 			if runtime.GOOS == "windows" {
 				return fmt.Errorf("currently not supported on windows")
 			}
-			var err error
-			if s, err = install.GetPid(); err != nil {
+
+			statuses, err := install.GetPid()
+			if err != nil {
 				return err
 			}
+			if len(statuses) != 0 {
+				for _, s := range statuses {
+					status := install.K0sStatus{
+						Pid:  s.Pid,
+						PPid: s.PPid,
+					}
+					if len(statuses) > 1 {
+						fmt.Println("=================")
```

Comment:
looks like some leftover debugs

Reply:
It's a separator for multiple items it seems. Don't know what happens with json/yaml.